bridge/contracts/Bridge.sol

```diff
@@ -392,8 +394,8 @@ contract Bridge is Initializable, IBridge, IERC777Recipient, UpgradablePausable,
         if(operator == address(this)) return; // Avoid loop from bridge calling to ERC77transferFrom
         require(to == address(this), "Bridge: Not to this address");
         address tokenToUse = _msgSender();
-        require(erc1820.getInterfaceImplementer(tokenToUse, _erc777Interface) != NULL_ADDRESS, "Bridge: Not ERC777 token");
-        require(userData.length != 0 || !from.isContract(), "Bridge: Specify receiver address in data");
+        require(ERC1820.getInterfaceImplementer(tokenToUse, _erc777Interface) != NULL_ADDRESS, "Bridge: Not ERC777 token");
+        require(userData.length != 0 || !from.isContract(), "Bridge: Receiver not in userData");
```

Comment:
maybe 'receiver address' instead of just 'receiver'?

Reply:
32+ characters